lib/rbi/rewriters/replace_attributes_with_methods.rb

```diff
@@ -0,0 +1,146 @@
+# typed: strict
+# frozen_string_literal: true
+
+module RBI
+  module TempHelpers
+    extend T::Sig
+  end
+
+  module Rewriters
+    class ReplaceAttributesWithMethods < Visitor
```

Comment:
Name's a bit wordy, and "Replace" is kind of implied by "Rewriters::".

How about:

```suggestion
    class AttrToMethods < Visitor
```

Reply:
Yeah I like it more, other rewriters always use an action verb 👍